WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -33,6 +33,7 @@ public class SiteUtils {
     public static final String GB_EDITOR_NAME = "gutenberg";
     public static final String AZTEC_EDITOR_NAME = "aztec";
     public static final String WP_STORIES_CREATOR_NAME = "wp_stories_creator";
+    public static final String WP_VIDEOPRESS_V5_JETPACK_VERSION = "8.5";
```

Comment:
Note, I chose this version number based on the Jetpack version we support for the Contact Info block. 

I couldn't find a definite version that the VideoPress v5 code was introduced on the web, and could find it supported as far back as Jetpack v7.2. I know the official stance is that we support "two releases prior to the current version", which is why I simply chose the oldest version we support for another block rather than going back to find a specific older version where web began supporting v5.

I'm open to changing this if others have feelings about what the version should be.